### DISCUSSION TITLE
Configure mediamtx to save only one file

### DISCUSSION THREAD

**Discussion by ErezAlster** [bluenviron/mediamtx#3854] [Questions]
### Question

Hi

I'm using mediamtx to stream an record video. for long recordings mediamtx is creating new video file every 1.4G.
Is there a parameter that allows to configure this setting?

eventually I want to have only one file if possible.
FYI, I'm recoding video for hours.

**Comment by aler9** [comment]
Hello, it's not possible to append recordings to an existing file but you can generate a single file that contains all of your (subsequent) recordings by using the playback server.
Concatenating files with gaps is not natively possible, but you can use FFmpeg to achieve that as described here: https://github.com/bluenviron/mediamtx/discussions/3243#discussioncomment-9551603
